ethcore/machine/src/tx_filter.rs

```diff
@@ -107,11 +107,12 @@ impl TransactionFilter {
 				match version_u64 {
 					2 => {
 						let (data, decoder) = transact_acl::functions::allowed_tx_types::call(sender, to, value);
-						client.call_contract(BlockId::Hash(*parent_hash), contract_address, data)
+						client.call_contract(BlockId::Hash(*parent_hash), CallOptions::new(contract_address, data))
+							.map_err(|e| e.to_string())
 							.and_then(|value| decoder.decode(&value).map_err(|e| e.to_string()))
 							.map(|(p, f)| (p.low_u32(), f))
 							.unwrap_or_else(|e| {
-								error!(target: "tx_filter", "Error calling tx permissions contract: {:?}", e);
```

Comment:
`{}` implies displaying less information than `{:?}`. Why was it changed?

Reply:
Can't tell exactly now from top of my head, but I think this formatting is used because we map error to string before, so debug fmt is not really needed here anymore. Will double check this when I'm with my laptop next week.